docs/contributing-to-airbyte/working-with-airbyte-platform.md

```diff
@@ -0,0 +1,77 @@
+---
+description: Working with Airbyte Platform
+---
+
+# Overview
+
+In order to facilitate faster development workflows, airbytehq/airbyte was split into two repositories on 2023-19-02:
+
+- airbytehq/airbyte connectors and development around
+- airbytehq/airbyte-platform the codebase that facilitates connectors
+- airbytehq/airbyte-protocol used by both connectors and platform, the underlying protocol that airbyte uses to perform data transfers
+
+If you have an existing pull request in airbytehq/airbyte that should instead target airbyte-platform there are two ways that you can move it over
+
+# Issues and Pull Requests
+
+Please open issues in airbytehq/airbyte and tag them platform. Issues have been disabled on airbyte-platform in order to keep issue tracking in one centralized location
+Please open pull requests targeting airbytehq/airbyte-platform in airbytehq/airbyte-platform and reference the issue in airbytehq/airbyte
+
+# Migrating from airbytehq/airbyte
+
+1. Using our /create-platform-pr [slash command tool](https://github.com/airbytehq/airbyte/blob/master/.github/workflows/create-oss-pr-snapshot.yml) for platform. Simply comment "create-platform-pr" on your existing airbytehq/airbyte pull request and let our automation perform the migration for you. Note that if there are any conflicts between files, for the purposes of simplicity the tool will always use your version of the file in the pull request vs the version that exists in airbyte-platform.
```

Comment:
Are these two separate processes (using the tool and manual migration)? If yes, this should be a bullet list instead of a numbered list